src/hex_http.erl

```diff
@@ -9,8 +9,9 @@
 -type status() :: non_neg_integer().
 -type headers() :: #{binary() => binary()}.
 -type body() :: nil.
+-type adapter_config() :: maps:map().
```

Comment:
out of curiosity, is there a difference between `map()` and `maps:map()`?

Reply:
Na. `maps:map()` probably is just `-type map() :: maps().`.

Reply:
actually there's no such type, let's go with just `map()` :)

```
iex> t :maps
@opaque iterator()
```